RELEASENOTES.md

```diff
@@ -2,10 +2,27 @@
 
 - It is now possible to skip the modification of dependency version numbers when running the Increment Version number workflow or the Create Release workflow
 
+### New Repository Settings
+
+- [`shortLivedArtifactsRetentionDays`](https://aka.ms/algosettings#shortLivedArtifactsRetentionDays) determines the number of days to keep short lived build artifacts (f.ex build artifacts from pull request builds, next minor or next major builds). 1 is default. 0 means use GitHub default.
+- [`longLivedArtifactsRetentionDays`](https://aka.ms/algosettings#longLivedArtifactsRetentionDays) determines the number of days to keep long lived build artifacts (f.ex build artifacts from CI/CD builds). 0 is the default and means use GitHub default.
+- [`preProcessorSymbols`](https://aka.ms/algosettings#preProcessorSymbols) is a list of preprocessor symbols to use when building the apps. This setting can be specified in workflow specific settings files or in conditional settings.
+- [`<buildMode>PreProcessorSymbols`](https://aka.ms/algosettings#cleanModePreProcessorSymbols) is a list of preprocessor symbols to be used when building apps in the \<buildMode> build mode. CleanModePreProcessorSymbols is a variation of this.
```

Comment:
Now that this PR also adds buildmode as a condition for conditional settings, shouldn't people just create a conditional setting called "preProcessorSymbols"? Similar to what you did here https://github.com/BusinessCentralApps/buildorder/blob/8e64503f243aaf6e24e7deca8d47b80975c97c54/BO-DK/.AL-Go/settings.json#L8

Reply:
I guess we could - but then we would have cleanModePreprocessorSymbols as a special case for compatibility?
Should we then issue a deprecation warning about that and ask people to modify this to a conditionalsetting instead?

Reply:
> Should we then issue a deprecation warning about that and ask people to modify this to a conditionalsetting instead?

That sounds like a good plan I think